_documentation/user/flags/jamming.md

```diff
@@ -5,4 +5,19 @@ type: Bad
 summary: Radar doesn't work. Can still see.
 ---
 
-...
+The Jamming flag causes static to block out radar, outside window viewing perspective is unaffected.
+
+## Tactics
+
+Jamming prevents any effective usage of radar, as it fills the radar with static.
+
+Countering the effects of Jamming requires good knowledge of the map and layout, along with effective moment patterns and positioning.
+If ricochet is enabled, one will not be able to notice shots coming from the rear, so avoiding areas where shots may ricochet into ones blind spot is recommended.
+The position where one is, will determine a lot of the effectiveness of Jamming.
```

Comment:
Either elaborate or remove this

Reply:
I'll revise lines 14-15, 16 will be removed.